@utahdts/utah-design-system/css/1-settings/_spacing.scss

```diff
@@ -4,7 +4,7 @@
 */
 
 @use "../1-settings/class-vars";
-
+// 4 8
```

Comment:
Is this comment intentional? what is its purpose?

Reply:
Oops